assets/js/amp-editor-blocks.js

```diff
@@ -231,25 +231,21 @@ var ampEditorBlocks = ( function() { // eslint-disable-line no-unused-vars
 				settings.attributes = {};
 			}
 			settings.attributes.ampFitText = {
-				type: 'boolean',
```

Comment:
Why remove all the types (just wondering) ?

Reply:
Apparently using `type` changed with Gutenberg 4.2.0 -- it's still allowed but not mandatory if the `source` is not specified, however, it's not supported if `source` is specified and the attribute won't work correctly (it will save but won't fetch the value as expected).

Reply:
Thanks, you may have just saved me some time/frustration in the future! 👍